supabase/migrations/20260410110000_harden_triage_ops_and_sla.sql

```
@@ -0,0 +1,77 @@
+    END IF;
+
+    UPDATE public.clinical_case_states
+    SET 
+        assigned_doctor_id = v_doctor_id,
+        triage_status = 'acknowledged',
+        acknowledged_at = now(),
+        updated_at = now()
+    WHERE id = p_case_state_id
+      AND (assigned_doctor_id IS NULL OR triage_status = 'pending');
+END;
+$$;
```

SQLFluff (4.1.0)

[error] Line 12, Position 12: Found unparsable section: 'IF NOT EXISTS clinical_vertical TEXT NOT...'

(PRS)